cssFiles/components/List/list.css

```diff
@@ -0,0 +1,18 @@
+.listSpaced {
+    padding: 0.5rem;
+    width: fit-content;
+}
+
+.list.non-bulletSpaced  {
+    list-style: none;
+}
+
+.list-item-inlineSpaced  {
+    display: flex;
```

Comment:
I'm not sure if this naming convention is good. Seems like you have mixed different formats. You can change this next time onwards